lib/datadog/appsec/contrib/faraday/patcher.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require_relative '../patcher'
+
+module Datadog
+  module AppSec
+    module Contrib
+      module Faraday
+        # Patcher for Faraday
+        module Patcher
+          include Datadog::AppSec::Contrib::Patcher
```

Comment:
I would suggest us to stop using an empty module

Reply:
oh wow, thanks for the tip, I actually haven't seen that it's empty

Reply:
@Strech btw, what do you say if we delete this module in a subsequent PR? (I mean I would still remove this iniclude from this PR of course)

Reply:
At first we can stop using it, yes, I have it cleaned, just need to open PR with that